frontend/src/pages/auth/LoginPage.tsx

```
@@ -43,6 +43,11 @@ const LoginPage: React.FC = () => {
   const [showMfaInput, setShowMfaInput] = useState(false);
   const navigate = useNavigate();
 
+  // Sync MFA input visibility with store state
+  useEffect(() => {
+    setShowMfaInput(mfaRequired);
+  }, [mfaRequired]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)